### ISSUE TITLE
A11y_Visual Studio Code_ Remove Terminal_ Keyboard: Not able to remove the Split Terminals using Keyboard.

### ISSUE THREAD

**Issue by Latha2564** [microsoft/vscode#131908]
GitHubTags:#A11y_VSCode;#A11ySev2;#A11yTCS;#A11yMAS;#DesktopApp;#Visual Studio Code Client;#WCAG2.1.1;#Keyboard;#MAC;#AINotInScope;#VoiceOver;#Benchmark-VisualStudioCode-MAC-Aug2021
### Environment Details:​
Application Name: Visual Studio Code
Standard Violated: Keyboard

### Repro Steps:​
1.Open Visual Studio Code editor
2.Tab till any Program code tab and press enter.
3.Tab till Terminal and press Enter.
4.  Split the Terminals and try to delete or remove the terminals .
5. Verify whether able to delete or remove the Split Terminals using Keyboard or not.
### Actual Result:
Not able to remove the Split Terminals using Keyboard

### Expected Result:
Users should  able to remove the Split Terminals using Keyboard

### User Impact:
Keyboard Users will find difficult in deleting the split terminals if the users are not able to remove the Split Terminals using Keyboard

**Comment by meganrogge** [comment]
In your gif, it doesn't look like any of your terminals are focused when you are trying to split or kill (or the cursor would be filled). I can't repro the issue.

![recording (62)](https://user-images.githubusercontent.com/29464607/131884739-c87ea033-9bb2-475c-b698-8a1f19f71703.gif)

**Comment by Latha2564** [comment]
![131908](https://user-images.githubusercontent.com/89508906/133747555-9d0e85cb-a549-4330-b802-fae465cfb1f0.gif)

**Comment by Tyriar** [comment]
@Latha2564 the gif is too low quality to understand
